src/python/espressomd/cellsystem.pyx

```diff
@@ -296,11 +296,15 @@ cdef class CellSystem(object):
                 Accuracy in skin to tune to.
         'int_steps' : :obj:`int`
                       Integration steps to time.
+        'adjust_max_skin' : :obj:`bool`, optional
+                            If ``True``, the value of ``max_skin`` is reduced
+                            to the maximum permissible skin (in case the passed
+                            value is too large). Set to ``False`` by default.
 
         Returns
         -------
         :attr:`skin`
 
         """
-        c_tune_skin(min_skin, max_skin, tol, int_steps)
+        c_tune_skin(min_skin, max_skin, tol, int_steps, int(adjust_max_skin))
```

Comment:
Why `int` here and in the signatures in `cellsystem.pxd` and `tuning.pxd` instead of `bool`?

Reply:
You're right, this is not necessary. Just a leftover from debugging I overlooked...